compiler/damlc/daml-compiler/src/DA/Daml/Compiler/DataDependencies.hs

```diff
@@ -479,6 +479,9 @@ generateSrcFromLf env = noLoc mod
 
         convConDetails :: LF.Type -> Gen (HsConDeclDetails GhcPs)
         convConDetails = \case
+            -- empty variant constructor (see issue #7207)
```

Comment:
"Empty" is such an overloaded word:
```suggestion
            -- variant constructor without argument (see issue #7207)
```
Or "payload" instead of "argument".

Reply:
Right, but "variant constructor without argument" is such a clumsy phrase, and what else could "empty variant constructor" mean?

Reply:
I guess the right answer here is "nullary variant constructor".

Reply:
I like "nullary variant constructor". 👍